packages/babel-plugin-transform-es2015-classes/test/fixtures/spec/super-reference-before-bare-super-inline/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "'super.*' is not allowed before super()"
```

Comment:
I think it's not valid because `super.foo` is being accessed before the call to `super()`. (See rule `SuperProperty:super.IdentifierName` in [spec](https://www.ecma-international.org/ecma-262/7.0/index.html#sec-super-keyword-runtime-semantics-)). 

That's consistent with behavior in Firefox. This will throw:

```js
class Bar {
  foo() {
    console.log('out');
  }
}
class Foo extends Bar {
  constructor() {
    super.foo(super());
  }
}
new Foo()
```
=> ReferenceError: |this| used uninitialized in Foo class constructor

But the following would run and print "out" in Firefox.

```js
class Bar {
  foo() {
    console.log('out');
  }
  get method() {
    return 'foo';
  }
}
class Foo extends Bar {
  constructor() {
    super[super().method]();
  }
}
new Foo()
```